CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ This CHANGELOG follows the format listed  [here](https://github.com/sensu-plugin
 ## [2.3.0] - 2018-02-09
 ### Changed
 - check-whois-domain-expiration.rb, check-whois-domain-expiration-multi.rb and check-jsonwhois-domain-expiration.rb: better resilience against errors, better reporting of erroneous responses (@DrMurx)
+- update whois-parser gem dependency to version 1.0.1 (@amdprophet)
```

Comment:
One gotcha - could you move this up to the Unreleased section? Or if you're comfortable releasing the plugin after, you can create the next release according to the community versioning practices here: https://github.com/sensu-plugins/community/blob/master/HOW_WE_VERSION.md